sdk-lib/src/main/java/cash/z/ecc/android/sdk/block/CompactBlockProcessor.kt

```diff
@@ -238,6 +237,13 @@ class CompactBlockProcessor internal constructor(
                         }
                         delay(napTime)
                     }
+                    is BlockProcessingResult.FailedDeleteBlockMetadata -> {
```

Comment:
I think we should be careful about our terminology, as I think we're deleting blocks from disk but not touching the metadata database table.